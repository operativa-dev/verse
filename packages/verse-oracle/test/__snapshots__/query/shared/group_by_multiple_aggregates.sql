-- Executing SQL: Parameters: []
select avg(t1.AlbumId) c0, count(*) c1
from Album t1
group by t1.ArtistId

