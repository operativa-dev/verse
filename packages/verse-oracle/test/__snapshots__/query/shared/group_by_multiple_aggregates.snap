[
  [
    21.5,
    2,
  ],
  [
    28,
    1,
  ],
  [
    81.5,
    2,
  ],
  [
    147,
    3,
  ],
  [
    84,
    1,
  ],
  [
    91,
    3,
  ],
  [
    104,
    21,
  ],
  [
    139.5,
    2,
  ],
  [
    142.5,
    2,
  ],
  [
    168,
    3,
  ],
  [
    176,
    1,
  ],
  [
    211,
    1,
  ],
  [
    243.5,
    4,
  ],
  [
    253,
    1,
  ],
  [
    254,
    1,
  ],
  [
    258,
    1,
  ],
  [
    294.5,
    2,
  ],
  [
    280,
    1,
  ],
  [
    281,
    1,
  ],
  [
    286,
    1,
  ],
  [
    294,
    1,
  ],
  [
    295,
    1,
  ],
  [
    302,
    1,
  ],
  [
    306,
    1,
  ],
  [
    317,
    1,
  ],
  [
    344,
    1,
  ],
  [
    12,
    1,
  ],
  [
    20,
    1,
  ],
  [
    118.85714285714286,
    14,
  ],
  [
    64,
    2,
  ],
  [
    51,
    1,
  ],
  [
    69.5,
    2,
  ],
  [
    119,
    1,
  ],
  [
    163.5,
    2,
  ],
  [
    175,
    1,
  ],
  [
    187,
    1,
  ],
  [
    206,
    1,
  ],
  [
    209.5,
    2,
  ],
  [
    212.5,
    2,
  ],
  [
    224.5,
    2,
  ],
  [
    252,
    1,
  ],
  [
    287,
    1,
  ],
  [
    293,
    1,
  ],
  [
    324,
    1,
  ],
  [
    325,
    1,
  ],
  [
    342,
    1,
  ],
  [
    21,
    2,
  ],
  [
    40,
    1,
  ],
  [
    47,
    1,
  ],
  [
    68,
    1,
  ],
  [
    117,
    3,
  ],
  [
    124,
    1,
  ],
  [
    125,
    1,
  ],
  [
    146,
    1,
  ],
  [
    162,
    1,
  ],
  [
    183,
    1,
  ],
  [
    214,
    1,
  ],
  [
    222.5,
    2,
  ],
  [
    226.5,
    2,
  ],
  [
    228,
    1,
  ],
  [
    268,
    1,
  ],
  [
    276,
    1,
  ],
  [
    278,
    1,
  ],
  [
    304,
    1,
  ],
  [
    326,
    1,
  ],
  [
    346,
    1,
  ],
  [
    347,
    1,
  ],
  [
    2.5,
    2,
  ],
  [
    14.5,
    2,
  ],
  [
    24.5,
    2,
  ],
  [
    39.75,
    4,
  ],
  [
    140.3,
    10,
  ],
  [
    42,
    1,
  ],
  [
    56.5,
    2,
  ],
  [
    71,
    1,
  ],
  [
    78,
    1,
  ],
  [
    80.5,
    4,
  ],
  [
    86,
    3,
  ],
  [
    120,
    1,
  ],
  [
    165,
    1,
  ],
  [
    186,
    6,
  ],
  [
    191,
    1,
  ],
  [
    203,
    1,
  ],
  [
    215,
    1,
  ],
  [
    241,
    1,
  ],
  [
    262,
    1,
  ],
  [
    264,
    1,
  ],
  [
    266,
    1,
  ],
  [
    279,
    1,
  ],
  [
    285,
    1,
  ],
  [
    315.3333333333333,
    3,
  ],
  [
    300,
    1,
  ],
  [
    301,
    1,
  ],
  [
    308,
    1,
  ],
  [
    318,
    1,
  ],
  [
    327,
    1,
  ],
  [
    335,
    1,
  ],
  [
    337,
    1,
  ],
  [
    19,
    1,
  ],
  [
    26.5,
    2,
  ],
  [
    31,
    1,
  ],
  [
    135.66666666666666,
    3,
  ],
  [
    84.66666666666667,
    3,
  ],
  [
    72.5,
    2,
  ],
  [
    115,
    1,
  ],
  [
    122,
    1,
  ],
  [
    160,
    1,
  ],
  [
    166,
    1,
  ],
  [
    192,
    1,
  ],
  [
    194,
    3,
  ],
  [
    199.5,
    2,
  ],
  [
    201.5,
    2,
  ],
  [
    217,
    3,
  ],
  [
    237.9,
    10,
  ],
  [
    247,
    1,
  ],
  [
    263,
    1,
  ],
  [
    270,
    1,
  ],
  [
    297,
    1,
  ],
  [
    309,
    1,
  ],
  [
    319,
    1,
  ],
  [
    328,
    1,
  ],
  [
    334,
    1,
  ],
  [
    339,
    1,
  ],
  [
    345,
    1,
  ],
  [
    6,
    1,
  ],
  [
    16.5,
    2,
  ],
  [
    121,
    2,
  ],
  [
    59.18181818181818,
    11,
  ],
  [
    52,
    1,
  ],
  [
    67,
    1,
  ],
  [
    75.5,
    4,
  ],
  [
    123,
    1,
  ],
  [
    144,
    1,
  ],
  [
    145,
    1,
  ],
  [
    177,
    1,
  ],
  [
    184,
    1,
  ],
  [
    246,
    1,
  ],
  [
    248,
    1,
  ],
  [
    257,
    1,
  ],
  [
    265,
    1,
  ],
  [
    269,
    1,
  ],
  [
    284,
    1,
  ],
  [
    290,
    1,
  ],
  [
    291,
    1,
  ],
  [
    298,
    1,
  ],
  [
    299,
    1,
  ],
  [
    303,
    1,
  ],
  [
    305,
    1,
  ],
  [
    311,
    2,
  ],
  [
    330,
    1,
  ],
  [
    331,
    1,
  ],
  [
    333,
    1,
  ],
  [
    5,
    1,
  ],
  [
    7,
    1,
  ],
  [
    9,
    1,
  ],
  [
    13,
    1,
  ],
  [
    18,
    1,
  ],
  [
    23,
    1,
  ],
  [
    41,
    1,
  ],
  [
    54.5,
    2,
  ],
  [
    83,
    1,
  ],
  [
    121,
    1,
  ],
  [
    147,
    1,
  ],
  [
    158.5,
    2,
  ],
  [
    180,
    5,
  ],
  [
    189,
    3,
  ],
  [
    196,
    1,
  ],
  [
    205,
    1,
  ],
  [
    207,
    1,
  ],
  [
    219.5,
    2,
  ],
  [
    250,
    3,
  ],
  [
    260,
    1,
  ],
  [
    275,
    1,
  ],
  [
    288,
    1,
  ],
  [
    289,
    1,
  ],
  [
    313,
    1,
  ],
  [
    324,
    3,
  ],
  [
    323,
    1,
  ],
  [
    329,
    1,
  ],
  [
    332,
    1,
  ],
  [
    340,
    1,
  ],
  [
    341,
    1,
  ],
  [
    2.5,
    2,
  ],
  [
    97.33333333333333,
    3,
  ],
  [
    33,
    1,
  ],
  [
    88,
    1,
  ],
  [
    93,
    1,
  ],
  [
    141,
    1,
  ],
  [
    161,
    1,
  ],
  [
    208,
    1,
  ],
  [
    221,
    1,
  ],
  [
    237.75,
    4,
  ],
  [
    259,
    1,
  ],
  [
    267,
    1,
  ],
  [
    272,
    1,
  ],
  [
    273,
    1,
  ],
  [
    277,
    1,
  ],
  [
    282,
    1,
  ],
  [
    283,
    1,
  ],
  [
    296,
    1,
  ],
  [
    307,
    1,
  ],
  [
    314,
    1,
  ],
  [
    321.5,
    2,
  ],
  [
    338,
    1,
  ],
]